datasets/Lab41-SRI-VOiCES.yaml

```diff
@@ -0,0 +1,29 @@
+Name: Voices Obscured in Complex Enrivonmental Settings (VOiCES)
```

Comment:
Please add a link to documentation.

https://travis-ci.org/awslabs/open-data-registry/builds/392915947#L494